src/review/views.py

```diff
@@ -468,17 +466,19 @@ def get_form_kwargs(self):
         return kwargs
 
     def get_context_data(self, **kwargs):
-        context = super(ReviewerDecisionEmail, self).get_context_data(**kwargs)
-        context['submission'] = get_object_or_404(
-            core_models.Book,
-            pk=self.kwargs['submission_id']
+        context = super(RequestedReviewerDecisionEmail, self).get_context_data(
+            **kwargs
+        )
+        context['heading'] = (
```

Comment:
At this point, instead of complaining about the full stop, I noticed that this sentence is copy / pasted at least 8 times in the PR. We should really store this piece of text in a dedicated file and source it when and where necessary.

Reply:
I've used four different variations on the same sentence, but I take your point. 

Do you think it would be worth adding settings to Rua for this? At the moment, there's no facility for automatically propagating settings across all Rua instances as far as I know, so it's quite laborious.